src/main/kotlin/com/team4099/robot2022/config/Constants.kt

```diff
@@ -54,10 +55,10 @@ object Constants {
     val DRIVETRAIN_WIDTH = 22.750.inches
 
     val DRIVE_SETPOINT_MAX = 15.feet.perSecond
-    val TURN_SETPOINT_MAX = 360.degrees.perSecond // TODO: Make sure this value is something good
+    val TURN_SETPOINT_MAX = 648.degrees.perSecond
 
-    val STEERING_VEL_MAX = 900.degrees.perSecond
-    val STEERING_ACCEL_MAX = 4500.degrees.perSecond.perSecond
+    val STEERING_VEL_NATIVE_MAX = 17_000.0
```

Comment:
can we convert this to real units and store it as that so it makes some degree of sense and can be transferred across robots or whatever